shell/browser/api/electron_api_menu.h

```diff
@@ -25,22 +25,32 @@ namespace electron::api {
 class BaseWindow;
 class WebFrameMain;
 
-class Menu : public gin_helper::DeprecatedWrappable<Menu>,
+class Menu : public gin::Wrappable<Menu>,
              public gin_helper::EventEmitterMixin<Menu>,
              public gin_helper::Constructible<Menu>,
              public gin_helper::Pinnable<Menu>,
```

Comment:
`gin_helper::Pinnable<Menu>` should be replaced with `gin_helper::SelfKeepAlive<T>`, you can refer api::session for example.